nav2_system_tests/src/system/test_system_node.py

```diff
@@ -14,15 +14,23 @@
 # limitations under the License.
 
 import math
+import os
+import random
 import sys
 import time
 
 from geometry_msgs.msg import Pose
 from geometry_msgs.msg import PoseStamped
 from geometry_msgs.msg import PoseWithCovarianceStamped
+
+import pandas
```

Comment:
If we need pandas, you should add the dependency to the package.xml. However, there is csv support in the Python standard library that you could potentially use as well.